bin/load_performance.py

```diff
@@ -2,139 +2,130 @@
-            WHEN it.severity = 'warning' AND rle.endpoint_end_date IS "" AND it.responsibility = 'internal' THEN 1
-            ELSE NULL
-        END
-        ) AS count_internal_warning,
-        COUNT(
-        CASE
-            WHEN it.severity = 'warning' AND rle.endpoint_end_date IS "" AND it.responsibility = 'external' THEN 1
-            ELSE NULL
-        END
-        ) AS count_external_warning,
-        COUNT(
-        CASE
-            WHEN it.severity = 'notice' AND rle.endpoint_end_date IS "" AND it.responsibility = 'internal' THEN 1
-            ELSE NULL
-        END
-        ) AS count_internal_notice,
-        COUNT(
-        CASE
-            WHEN it.severity = 'notice' AND rle.endpoint_end_date IS "" AND it.responsibility = 'external' THEN 1
-            ELSE NULL
-        END
-        ) AS count_external_notice
-    FROM
-        provision p
-        LEFT JOIN organisation o ON o.organisation = p.organisation
-        LEFT JOIN rle_with_latest rle
-            ON REPLACE(rle.organisation, '-eng', '') = p.organisation
-            AND rle.pipeline = p.dataset
-        LEFT JOIN issue i 
-            ON rle.resource = i.resource AND rle.pipeline = i.dataset
-        LEFT JOIN issue_type it 
-            ON i.issue_type = it.issue_type AND it.severity != 'info'
-    GROUP BY
-        p.organisation,
-        o.name,
-        p.dataset
-    ORDER BY
-        p.organisation,
-        o.name;
-    """)
-    data = cursor.fetchall()
-    conn.close()
-    return data
+    query = """
+        select  
+        count(*) as count_issues, strftime('%d-%m-%Y', 'now') as date,
+        i.issue_type as issue_type, it.severity, it.responsibility, i.dataset, i.resource
+        from issue i
+        inner join resource r on i.resource = r.resource
+        inner join issue_type it on i.issue_type = it.issue_type
+        where r.end_date = ''
+        group by i.dataset,i.resource,i.issue_type
+    """
+    df_issue = pd.read_sql_query(query, conn)
+    return df_issue
 
-def create_organisation_dataset_summary(data, performance_db_path):
+def create_performance_tables(merged_data, performance_db_path):
     conn = sqlite3.connect(performance_db_path)
-    cursor = conn.cursor()
-    
-    cursor.execute("""
-        CREATE TABLE IF NOT EXISTS provision_summary (
-            organisation TEXT,
-            name TEXT,
-            dataset TEXT,
-            active_endpoint_count INT,
-            error_endpoint_count INT,
-            count_internal_error INT,
-            count_external_error INT,
-            count_internal_warning INT,
-            count_external_warning INT,
-            count_internal_notice INT,
-            count_external_notice INT
+    issue_table_name = "issue_summary"  
+
+    issue_data_filtered = merged_data[merged_data['endpoint'].notna()]
+    issue_data_filtered.to_sql(issue_table_name, conn, if_exists='replace', index=False)
+    final_result = merged_data.groupby(['organisation', 'name', 'dataset']).agg(
```

Comment:
should this not be using the `issue_data_filtered` instead of `merged_data` if it's the final result? and the same for the following aggregations